grails-app/services/streama/DefaultDataService.groovy

```diff
@@ -27,7 +27,7 @@ class DefaultDataService {
             password: 'admin',
             fullName: 'Administrator',
             enabled: true,
-            roles: [Role.findByAuthority("ROLE_ADMIN"), Role.findByAuthority("ROLE_CONTENT_MANAGER"),Role.findByAuthority(authority: "ROLE_TRUSTED_USER")]
+            roles: [Role.findByAuthority("ROLE_ADMIN"), Role.findByAuthority("ROLE_CONTENT_MANAGER"),Role.findByAuthority( "ROLE_TRUSTED_USER")]
```

Comment:
Because this is a crucial change that keeps people from running the project locally, I think it's good to separate that into another PR. This PR looks like it's still going to take a while to be released. If you want I can submit a PR with this change.